tests/unit_tests/test_solver.cpp

```diff
@@ -0,0 +1,59 @@
+#include "gtest/gtest.h"
+
+#include <Kokkos_Core.hpp>
+
+#include "src/rigid_pendulum_poc/solver.h"
+
+namespace oturb_tests {
```

Comment:
perhaps this should be `openturbine_tests`?  We won't be calling this namespace much, if at all, so it can be longer without sacrificing line length.